apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx

```diff
@@ -18,8 +18,8 @@ import { useIsMobileWindowWidth } from '~/hooks/useWindowSize';
 import GlobeIcon from '~/icons/GlobeIcon';
 import ShareIcon from '~/icons/ShareIcon';
 import { useTrack } from '~/shared/contexts/AnalyticsContext';
+import { replaceUrlsWithMarkdownFormat } from '~/shared/utils/replaceUrlsWithMarkdownFormat';
 import { getExternalAddressLink } from '~/shared/utils/wallet';
-import formatUrl from '~/utils/formatUrl';
```

Comment:
basically I realised that this formatUrl function I added in an earlier PR doesn't actually handle all the cases correctly and updated to use the new function and tested that this works. Before, it would fail the case of the description being already formatted in markdown.
<img width="1162" alt="Screenshot 2023-09-20 at 8 19 13 AM" src="https://github.com/gallery-so/gallery/assets/49758803/e960efb7-d72b-4b04-b0fd-48d53bb8d4cf">
<img width="520" alt="Screenshot 2023-09-20 at 8 19 17 AM" src="https://github.com/gallery-so/gallery/assets/49758803/8412b75d-d8c4-4376-a0f2-5dc379d906d4">